shepherd.js/src/utils/floating-ui.ts

```diff
@@ -211,8 +211,16 @@ export function getFloatingUIOptions(
     );
 
     if (arrowEl) {
+      const arrowOptions =
+        typeof step.options.arrow === 'object'
+          ? step.options.arrow
+          : { padding: 4 };
+
       options.middleware.push(
-        arrow({ element: arrowEl, padding: hasEdgeAlignment ? 4 : 0 })
+        arrow({
+          element: arrowEl,
+          padding: hasEdgeAlignment ? arrowOptions.padding : 0
```

Comment:
```suggestion
          padding: hasEdgeAlignment ? step.options.arrow?.padding ?? 4 : 0
```

Reply:
I think this would be working javascript (I.e., `(true).padding` is `undefined`) but typescript doesn't allow that.

Reply:
Oh, gotcha. Didn't realize TS wouldn't allow it.